spec/features/earning_and_losing_points_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+# frozen_string_literal: true
+RSpec.describe 'Kudo spec', type: :feature do
+  let!(:kudo) { build(:kudo) }
+  let!(:company_value) { create(:company_value) }
+
+  before do
+    login_as kudo.giver, scope: :employee
+    visit root_path
+  end
+
+  it 'gives point for earned kudos and reduces for lost kudo' do
+    # creating new kudo for receiver
+    click_link 'New Kudo'
+    fill_in 'kudo[title]', with: kudo.title
+    fill_in 'kudo[content]', with: kudo.content
+    select kudo.receiver.email, from: 'kudo[receiver_id]'
+    select company_value.title, from: 'kudo[company_value_id]'
+    click_button 'Create Kudo'
+    expect(page).to have_content 'Kudo was successfully created.'
+    kudo.giver.reload
+
+    # signing in as receiver and checking available points
+    click_link 'Sign Out'
+    expect(page).to have_content 'You need to sign in or sign up before continuing.'
```

Comment:
do we really need to check it here ;) We should focus on checking available points in this spec, since the login is tested anyways in dedicated specs

Reply:
Thanks for this suggestion - I added new session for receiver to check his points :)